tasks/theo-agent.yml

```diff
@@ -27,6 +27,26 @@
     theo_agent_config_public_key_path: ""
   when: not theo_agent_verify_signature|bool
 
+- name: Set hostname prefix
+  set_fact:
+    theo_agent_config_hostname_prefix: "hostname-prefix: {{ theo_agent_hostname_prefix }}"
+  when: theo_agent_hostname_prefix | length > 0
+
+- name: Set hostname prefix
+  set_fact:
+    theo_agent_config_hostname_prefix: ""
+  when: theo_agent_hostname_prefix | length == 0
```

Comment:
I think this can be replaced with some sort of conditional within the template, kinda like:

```
{% if theo_agent_hostname_prefix %}hostname-prefix: {{ theo_agent_hostname_prefix }}{% endif %}
```

Reply:
done as suggested